pyproject.toml

```
@@ -23,6 +23,8 @@ optional = true
 [tool.poetry.group.dev.dependencies]
 pytest = "^8.3.2"
 ruff = "^0.5.6"
+sphinx = "^7.4.7"
+sphinx-autobuild = "^2024.4.16"
```

```suggestion

[tool.poetry.group.docs]
optional = true

[tool.poetry.group.docs.dependencies]
sphinx = "^7.4.7"
sphinx-autobuild = "^2024.4.16"
```

Let's keep docs deps separate

---

Reasonable